source/includes/usage-examples/write-code-examples.php

```diff
@@ -0,0 +1,104 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+$collection = $client->db->coll;
+
+// Inserts one document that stores the specified value
+// start-insert-one
+$result = $collection->insertOne(['<field name>' => '<value>']);
+// end-insert-one
+
+// Inserts multiple documents that store the specified values
+// start-insert-multiple
+$result = $collection->insertMany(
+    ['<field name>' => '<value>'],
+    ['<field name>' => '<value>'],
+);
+// end-insert-multiple
+
+// Updates a document that matches the specified criteria
+// start-update-one
+$result = $collection->updateOne(
+    ['<field to match>' => '<value to match>'],
+    ['$set' => ['<field name>' => '<value>']],
+);
+// end-update-one
+
+// Updates all documents that match the specified criteria
+// start-update-multiple
+$result = $collection->updateMany(
+    ['<field to match>' => '<value to match>'],
+    ['$set' => ['<field name>' => '<value>']],
+);
+// end-update-multiple
+
+// start-replace-one
+$result = $collection->replaceOne(
+    ['<field to match>' => '<value to match>'],
+    [
+        '<first new field>' => '<value>',
+        '<second new field>' => '<value>',
+    ],
+);
+// end-replace-one
+
+// Deletes a document that matches the specified criteria
+// start-delete-one
+$result = $collection->deleteOne(['<field name>' => '<value>']);
+// end-delete-one
+
+// Deletes all documents that match the specified criteria
+// start-delete-multiple
+$result = $collection->deleteMany(['<field name>' => '<value>']);
+// end-delete-multiple
+
+// Runs a bulk operation based on the instructions in each array entry
+// start-bulk-write
+$result = $collection->bulkWrite(
+    [
+        [
+            'insertOne' => [
+                ['<field name>' => '<value>'], 
+            ],
+        ],
+        [
+            'replaceOne' => [
+                ['<field to match>' => '<value to match>'],
+                [
+                    '<first new field>' => '<value>',
+                    '<second new field>' => '<value>',
```

Comment:
S: double check if we have a convention around spelling out ordinals

Reply:
I couldn't find anything specific, but changed to numerals based on other landing pages